python/ray/client_builder.py

```diff
@@ -85,7 +100,9 @@ def _get_builder_from_address(address: Optional[str]) -> ClientBuilder:
         return _LocalClientBuilder(None)
     if address is None:
         try:
-            with open("/tmp/ray/current_cluster", "r") as f:
+            cluster_file = os.path.join(ray._private.utils.get_user_temp_dir(),
```

Comment:
```suggestion
```suggestion
			# NOTE: This is not scoped to the `Node::get_temp_dir_path`, because
			# this file is accessed before the `Node` object is not yet created.
            cluster_file = os.path.join(ray._private.utils.get_user_temp_dir(),
```